other/lru_cache.py

```diff
@@ -54,58 +60,70 @@ class Lru_Cache:
 
     >>> cache.set(3, 3)
 
-    >>> cache.get(2)
-    Traceback (most recent call last):
-    ...
-    ValueError: Key '2' not found in cache
+    >>> cache.get(2)    # None returned
```

Comment:
Why get rid of the exception?
Now the caller is blind to whether 2 was in cache and set to None vs. 2 was not in cache.

Reply:
Its given in the type hint, the function will return an integer or None if the key is absent